src/sentry/search/utils.py

```diff
@@ -427,6 +427,10 @@ def parse_query(projects, query, user, environments):
                     results["unassigned"] = True
                 elif value == "assigned":
                     results["unassigned"] = False
+                elif value == "triaged":
+                    results["triaged"] = True
+                elif value == "untriaged":
+                    results["triaged"] = False
```

Comment:
I believe this is only used for event search. By specifying these we get the desired error message:
![image](https://user-images.githubusercontent.com/549473/97155225-41857100-1732-11eb-8049-12602b85ac9e.png)
(instead of it complaining about unknown "is" attributes)